src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -99,7 +99,15 @@
      */
     @Test
     public void handlesSocket() throws IOException {
-        final Socket socket = createMockSocket();
+        final Socket socket = createMockSocket(
```

Comment:
@exper0 from my point of view, the issue is pointless, unless you could get rid of mocking and introduce fake Stream as it was suggested by Yegor